src/questions.js

```diff
@@ -0,0 +1,103 @@
+//questions that will be prompted to the user to build team
+const teamBuilderQuestions = [
```

Comment:
Please add validation to all these questions to ensure 1) They cannot be left blank and 2) That the answer meets the format you expect.